src/views/imaged-object-editor/imaged-object-editor.vue

```diff
@@ -496,7 +496,10 @@ export default class ImagedObjectEditor
             // The band is caused by the new width (old height) being smaller than the old width.
             // There actually two bands, one to the left and one to the right. The right one can't be seen.
             const bandWidth = (this.imageWidth - this.imageHeight) / 2;
-            translate = `translate(-${bandWidth}, ${bandWidth})`;
+
+            // Take absolute value, thus preventing a case of negative values turning into non numbers, e.g. --1878.278443113… for IQ7
+            translate = `translate(-${Math.abs(bandWidth)}, ${bandWidth})`;
```

Comment:
But if bandwidth is negative, say -17, we will get a translate of(-17, -17). Is this intentional? A better fix would be:

`translate(${-bandwidth}, ${bandwidth})`

It is possible to put expressions in ${}

Reply:
All the values I checked were positive, only one was negative(1Q7). but the former solution produced --1800, which caused an exception: vue.runtime.esmError: <g> attribute transform: Expected number, "…(0.1) translate(--1878.278443113…".
translate = `translate(${(-bandWidth)}, ${bandWidth})`; acts ok.
I thought the reason was to make all x values negative, thus theis solution:
translate = `translate(-${Math.abs(bandWidth)}, ${bandWidth})`;
but they act the same.

Reply:
Use my suggestion, it is better and works with all numbers.